### PR TITLE
adjustments/develop : styling

### PR DESCRIPTION
**Pull request by KianMiridoozini**
Most of the changes consist  styling and UX enhancements

Styling (focus on mobile displays but also some general desktop optimizations):
- Profile
- Profile edit form
- Admin dashboard

### Fixed
checkout  "adjustments/develop : updated"  commit for more info

Also merged with latest develop

### CODE REVIEW

File: app/views/user/form.php [hunk 1]
```diff
@@ -67,7 +72,4 @@
         </div>
         <button type="submit" class="btn btn-success">Save Changes</button>
     </form>
```
Comment (jayb2302) [inline app/views/user/form.php:74]: ooowww I had a very good argument for keeping it down

File: app/views/user/profile.php [hunk 2]
```diff
@@ -24,9 +24,21 @@
...
+                    <?php if (isset($isFollowing) && $isFollowing): ?>
+                        <form action="/user/unfollow/<?= htmlspecialchars($profileUserId); ?>" method="POST">
+                            <button type="submit" class="btn btn-danger unfollow">Unfollow</button>
+                        </form>
+                    <?php else: ?>
+                        <form action="/user/follow/<?= htmlspecialchars($profileUserId); ?>" method="POST">
+                            <button type="submit" class="btn btn-primary follow">Follow</button>
+                        </form>
+                    <?php endif; ?>
+                <?php endif; ?>
```
Comment (jayb2302) [inline app/views/user/profile.php]: Why did you move it to the top again ?

Comment (KianMiridoozini) [reply app/views/user/profile.php]: Makes more sense as the button feels so out of place elsewhere.

You'll see when you see the changes

File: app/views/user/profile.php [hunk 3]
```diff
@@ -38,19 +50,9 @@ class="profile-img">
...
-                        </form>
-                    <?php else: ?>
-                        <form action="/user/follow/<?= htmlspecialchars($profileUserId); ?>" method="POST">
-                            <button type="submit" class="btn btn-primary">Follow</button>
-                        </form>
-                    <?php endif; ?>
-                <?php endif; ?>
                 <div class="follow-stats">
                     <small>Following: <?= htmlspecialchars($userProfile->getFollowingCount() ?? 0); ?></small>
+                    <i>|</i>
```
Comment (jayb2302) [inline app/views/user/profile.php]: what's this i tag doing?

Comment (KianMiridoozini) [reply app/views/user/profile.php]: Separating the following and follewers by |

Following:12 | Followers:34

File: public/assets/css/components/user-management.css [hunk 4]
```diff
@@ -90,3 +90,106 @@
...
+    color: var(--text-secondary);
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+  }
+
+  .update-status-form .button:hover {
+    background: var(--button-hover);
+  }
+}
```
Comment (jayb2302) [inline public/assets/css/components/user-management.css:140]: no nesting?

Comment (KianMiridoozini) [reply public/assets/css/components/user-management.css:140]: It was 4 am, the nesting was being a pain, so I did it without nesting for now

File: app/views/user/profile.php (hunk 3 above)
Comment (jayb2302) [reply app/views/user/profile.php]: it doesnt need to be in a icon tag

File: public/assets/css/components/user-management.css (hunk 4 above)
Comment (jayb2302) [reply public/assets/css/components/user-management.css:140]: What happened to the nesting?

File: app/views/user/profile.php (hunk 3 above)
Comment (KianMiridoozini) [reply app/views/user/profile.php]: I'm blind, I thought you wrote "what is this | doing". totally misread it
Fixed: I have changed it do divs instead. Yes it doesn't need that either, but for the sake of it not blending with other vertical lines in VS Code, I decided to keep it in div tags

File: public/assets/css/components/user-management.css (hunk 4 above)
Comment (KianMiridoozini) [reply public/assets/css/components/user-management.css:140]: fixed

**Review by jayb2302 — APPROVED** [review]
Approved
